src/lib.rs

```diff
@@ -46,6 +46,19 @@ use ffmpeg_next as ffmpeg;
 
 const CRATE_ENV_VAR: &str = "TWENTY_TWENTY";
 
+/// The different modes available for the TWENTY_TWENTY environment variable.
+#[derive(PartialEq)]
+enum Mode {
+    /// Only assert the image diff is within the given threshold.
+    Default,
```

Comment:
I suggest deriving Default (and put `#[default]` on this variant, so the derive knows which variant to use as the default